code/modules/clothing/spacesuits/plasmamen.dm

```diff
@@ -64,9 +65,12 @@
 	visor_toggling()
 	update_icon()
 
-/obj/item/clothing/head/helmet/space/plasmaman/AltClick(mob/user)
-	if(user.canUseTopic(src, BE_CLOSE))
-		toggle_welding_screen(user)
+/obj/item/clothing/head/helmet/space/plasmaman/examine()
+	. = ..()
+	if(attached_hat)
+		. += "<span class='notice'>There's [attached_hat.name] placed on the helmet. Alt+Left click to remove it.</span>"
```

Comment:
there also isn't an "a"/"an" here, but that's a bit more complicated to fix, so I didn't bring it up at first

Reply:
why does language have to be so complicated

Reply:
we could also just completely ignore this like I did at first, cause it's in the examine text when you have a hat on, which might be not done that often

Reply:
there's also this alternative, but i'm not sure if we want that tone or voice, cause it's talking from the perspective of the hat
```suggestion
		. += "<span class='notice'>The [attached_hat.name] is placed on the helmet. Alt+Left click to remove it.</span>"
```

Reply:
Grammar in SS13 is just not so well made. Thanks for checking on the minor things though, I truly appreciate it.